### PR TITLE
Analytic Feeds user guide notebooks

### PR DESCRIPTION
**Pull request by cameronbronstein**
Add two notebooks under `./jupyter-notebooks/analytics/user-guide/`.
Add additional RUN command to Dockerfile to `pip install` additional dependencies required for these notebooks.

### CODE REVIEW

File: jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb [hunk 1]
```diff
@@ -0,0 +1,1546 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make a request to get the subscription's mosaics\n",
+    "subscription_mosaics = requests.get(target_mosaic_series['_links']['mosaics'], auth=BASIC_AUTH).json()\n",
```
Comment (strixcuriosus) [inline jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb:1169]: The subscription mosaics will be the subset of these within the subscription time range of interest. To make the map below avoid showing months for which there are not results, you can filter the mosaics list here.
e.g.
```python
target_mosaics = requests.get(target_mosaic_series['_links']['mosaics'], auth=BASIC_AUTH).json()['mosaics']
subscription_mosaics = [m for m in target_mosaics if m['first_acquired'] >= subscription['startTime'] and m['last_acquired'] <= subscription['endTime']]
```

File: jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb [hunk 2]
```diff
@@ -0,0 +1,1546 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We only need the year, month, and day for monthly Basemaps\n",
+    "start_time = seg_subscription['startTime'].split('T')[0]\n",
+    "end_time = seg_subscription['endTime'].split('T')[0]\n",
+    "\n",
+    "print('Subscription startTime: {}'.format(start_time))\n",
+    "print('Subscription startTime: {}'.format(end_time))"
```
Comment (strixcuriosus) [inline jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb:1088]: should say Subscription endTime

File: jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb (hunk 1 above)
Comment (strixcuriosus) [reply jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb:1169]: Ah, I see you have a filtering example in a later section. It still might be nice to constrain what gets visualized in the first raster data demo map.

**Comment by strixcuriosus** [comment]
Thanks for reviving and improving these user guide notebooks, @cameronbronstein!

**Comment by strixcuriosus** [comment]
Since these are the user guide companion notebooks, let's include a link to the user guide somwhere:
https://developers.planet.com/docs/analytics/

**Comment by cameronbronstein** [comment]
Latest commits reflect changes made based on Ash's feedback.  
Also added additional case study notebook.

**Comment by strixcuriosus** [comment]
LGTM, thanks for making these improvements and adding a new demo notebook @cameronbronstein
Let's see if @sarasafavi has any additional comments.

Follow up TODO: make sure we update the docker image published to dockerhub at https://hub.docker.com/r/planetlabs/notebook

File: planet-notebook-docker/Dockerfile [hunk 3]
```diff
@@ -18,6 +18,11 @@ USER $NB_UID
 RUN conda config --set channel_priority strict && \
     conda install -y -c conda-forge --file /tmp/requirements.txt
 
+# install the following libraries w/ pip to support analytics/user-guide nb's
+# build fails when these dependencies are added to requirements/ installed with conda
+# python -m pip as per https://github.com/pypa/pip/issues/5599
+RUN python -m pip install descartes holoviews geoviews xarray datashader
```
Comment (jreiberkyle) [inline planet-notebook-docker/Dockerfile:24]: I'm curious how the build fails. Is one of the packages not available or is it unable to resolve the dependencies? Also, if any of these packages don't cause a build failure, could you put them in conda? We are trying to keep all the libraries installed in conda to minimize conflicts as much as possible.

Comment (cameronbronstein) [reply planet-notebook-docker/Dockerfile]: Upon revisiting this, I was able to slim down these dependencies and add them to requirements.txt. By pinning the requirements, the conda build does not get `killed`.

**Comment by jreiberkyle** [comment]
This looks good to me. @SaraPlanet I ran out of time to build the docker image and upload it. Can you take that on?

**Comment by SaraPlanet** [comment]
Did you mean @sarasafavi ?

**Comment by sarasafavi** [comment]
@SaraPlanet yes she did, sorry for the ping :) 

@jreiberkyle can do, thanks again for reviewing!

**Comment by jreiberkyle** [comment]
@cameronbronstein and @strixcuriosus are you ready to merge this?

**Comment by jreiberkyle** [comment]
Updated the [dockerhub image](https://hub.docker.com/r/planetlabs/notebooks). Thanks for your hard work on this @cameronbronstein !

**Pull request merged**